docs/API.md

```diff
@@ -1591,7 +1661,7 @@ whose series is 2. Available filters are:
 * promulgated - the charm has been promulgated.
 * provides - interfaces provided by the charm.
 * requires - interfaces required by the charm.
-* series - the charm's series.
+* series - the set of series associated with the charm.
```

Comment:
It seems that this is something that should be added when it's needed though. It's not been needed to date, and it seems that to find a series supported for a charm there'd be better methods vs a elasticsearch search. I'm with @rogpeppe that until something comes up that requires it directly that we don't go changing things on the chance they might be needed.